packages/create-next-app/templates/default-tw-empty/ts/pages/index.tsx

```diff
@@ -1,7 +1,17 @@
+import Head from "next/head";
+
 export default function Home() {
   return (
-    <main>
-      <div>Hello world!</div>
-    </main>
+    <>
+      <Head>
+        <title>Create Next App</title>
+        <meta name="description" content="Generated by create next app" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="icon" href="/favicon.ico" />
```

Comment:
don't think we should have this if we removed the favicon

Reply:
@ztanner Wouldn't they have to add the line back though? They would just have to provide their own `favicon.ico`